infoschema/tables.go

```diff
@@ -1611,6 +1647,8 @@ func (it *infoschemaTable) getRows(ctx sessionctx.Context, cols []*table.Column)
 		fullRows, err = dataForSlowLog(ctx)
 	case tableTiDBHotRegions:
 		fullRows, err = dataForTiDBHotRegions(ctx)
+	case tableAnalyzeStatus:
+		fullRows = dataForAnalyzeStatus()
```

Comment:
how about sort the rows by status and update time in descending order? so that if there a lot of analyze job infos which takes more than a screen to display, we can see the running jobs in the tail results.